app/processing/delta/ensure-value-consistency.js

```diff
@@ -1,7 +1,7 @@
 const ensureValueConsistency = (paymentRequest) => {
   // ensure no value gained or lost. if variation between total and lines apply difference to first gross line
   const invoiceLineTotal = getInvoiceLineTotal(paymentRequest)
-  const variation = Math.abs(paymentRequest.value) - Math.abs(invoiceLineTotal)
+  const variation = paymentRequest.value - invoiceLineTotal
```

Comment:
Should not be working with absolute values as will result in incorrect variance calculation